sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/collectionOperations.scala

```diff
@@ -26,6 +26,9 @@ import org.apache.spark.sql.types._
 /**
  * Given an array or map, returns its size.
  */
+@ExpressionDescription(
+  usage = "_FUNC_(column) - Returns the length of the array or map stored in the column",
+  extended = "> SELECT _FUNC_(array(\"b\", \"d\", \"c\", \"a\"));\n 4")
```

Comment:
Tried to do it, but realized annotation argument needs to be a constant. A newline marker `\n` is an issue here. We are unable to call `stripMargin` after triple quotes. Thus, I called `stripMargin` when building `ExpressionInfo`. Do you like it?